### PR TITLE
Add django upgrade jobs for a11y, js, quality

### PR DESCRIPTION
**Pull request by michaelyoungstrom**

### CODE REVIEW

File: platform/jobs/edxPlatformAccessibilityPr.groovy [hunk 1]
```diff
@@ -28,77 +28,130 @@ catch (any) {
...
+    workerLabel: 'jenkins-worker',
+    whitelistBranchRegex: /^((?!open-release\/).)*$/,
+    context: 'jenkins/a11y',
+    triggerPhrase: /.*jenkins\W+run\W+a11y.*/
+]
+
+Map django19JobConfig = [
+    open : true,
+    jobName : 'edx-platform-django-1.9-accessibility-pr',
+    subsetJob: 'edx-platform-test-subset',
```
Comment (estute) [inline platform/jobs/edxPlatformAccessibilityPr.groovy]: A11y doesn't have subset jobs.

File: platform/jobs/edxPlatformJsPr.groovy [hunk 2]
```diff
@@ -36,78 +36,131 @@ catch (any) {
...
+    workerLabel: 'jenkins-worker',
+    whitelistBranchRegex: /^((?!open-release\/).)*$/,
+    context: 'jenkins/js',
+    triggerPhrase: /.*jenkins\W+run\W+js.*/
+]
+
+Map django19JobConfig = [
+    open : true,
+    jobName : 'edx-platform-django-1.9-js-pr',
+    subsetJob: 'edx-platform-test-subset',
```
Comment (estute) [inline platform/jobs/edxPlatformJsPr.groovy]: JS doesn't have subset jobs.

File: platform/jobs/edxPlatformQualityPr.groovy [hunk 3]
```diff
@@ -56,6 +56,48 @@ Map publicJobConfig = [
...
+    workerLabel: 'django-upgrade-worker',
+    whitelistBranchRegex: /^((?!open-release\/).)*$/,
+    context: 'jenkins/django-1.10/quality',
+    triggerPhrase: /.*jenkins\W+run\W+django110\W+quality.*/,
+    defaultTestengBranch: 'master',
+    commentOnly: true,
+    djangoVersion: '1.10'
+]
+
+Map django111JobConfig = [
```
Comment (estute) [inline platform/jobs/edxPlatformQualityPr.groovy]: Should we create a special flow-worker, similar to what you did here: https://github.com/edx/jenkins-job-dsl/pull/312 ?

Comment (michaelyoungstrom) [reply platform/jobs/edxPlatformQualityPr.groovy]: Not a bad idea. since we're just running them by comment lets keep it as is for now.

**Pull request merged**